Umbraco Commerce v14-rc3 release notes (#6342)
umbraco/UmbracoDocs: 14/umbraco-commerce/release-notes/README.md

This section contains the release notes for Umbraco Commerce 14 including all changes for this version.

#### 14.0.0-rc3 (12th August 2024)

> * Added warning on store dashboard / analytics section if the store has multiple currencies by no currency exchange rate service configured.

Reviewer: ```suggestion
* Added warning on store dashboard/analytics section if the store has multiple currencies when no currency exchange rate service is configured.
```

> * Added nullable type support to payment provider / shipping provider settings models.

Reviewer: ```suggestion
* Added nullable type support to payment provider/shipping provider settings models.
```

* Added `UcStoreContext` to exported NPM package.

> * Added background task licenses resolver to allow Umrbaco Commerce to run in background tasks without erroring.

Reviewer: ```suggestion
* Added background task licenses resolver to allow Umbraco Commerce to run background tasks without error.
```

* Added delete support to carts and orders.
* Updated store create dialog to redirect to store editor on create.
* Fixed error on store dashboard when using SQL Server due to group by issue in SQL statement [#547](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/547).

> * Fixed errors in v14 migrations when using a seperate table for Umbraco Commerce.

Reviewer: ```suggestion
* Fixed errors in v14 migrations when using a separate table for Umbraco Commerce.
```

* Fixed error in v14 migrations where data types are not correctly migrated.
* Fixed store tree not updating when a new store is created.
* Fixed shipping provider advanced settings now showing in an advanced dropdown editor UI.

> * Fixed broken localzation keys.

Reviewer: ```suggestion
* Fixed broken localization keys.
```
